openedx/core/djangoapps/programs/tests/test_tasks.py

```diff
@@ -0,0 +1,244 @@
+"""
+Tests for programs celery tasks.
+"""
+
+import ddt
+from django.test import TestCase
+from edx_rest_api_client.client import EdxRestApiClient
+import httpretty
+import json
+import mock
+
+from oauth2_provider.tests.factories import ClientFactory
+from student.tests.factories import UserFactory
+
+from openedx.core.djangoapps.credentials.tests.mixins import CredentialsApiConfigMixin
+from openedx.core.djangoapps.programs.tests.mixins import ProgramsApiConfigMixin
+from openedx.core.djangoapps.programs import tasks
```

Comment:
Nitpick: I think the most recent version of Pylint would flag this block of imports. I would separate standard library, third-party, and first-party imports into separate groups.